doc/users/next_whats_new/image_interpolation_rgbastage.rst

```diff
@@ -0,0 +1,23 @@
+Image interpolation now possible at RGBA stage
+----------------------------------------------
+
+Images in Matplotlib created via `~.axes.Axes.imshow` are resampled to match 
+the resolution of the current canvas.  It is useful to apply an anto-aliasing
+filter when downsampling to reduce Moire effects.  Previously this was 
+done at the data stage, so adjoining data values would be interpolated 
+between, and then the data would be passed to the normalization, and then 
```

Comment:
'interpolated between' is a bit awkward, due to the dangling preposition. I'm not fully happy with it, but maybe 'interpolated together'?

Could also drop 'and then the data would be'.
```suggestion
done at the data stage, so adjoining data values would be interpolated 
together, passed to the normalization, and finally to 
```

Reply:
Changed to 

```
By default, interpolation 
is done on the data, a norm applied, and then the colormapping performed. 
```